arcgis-ios-sdk-samples/Utility Network/Display utility association/DisplayUtilityAssociationViewController.swift

```diff
@@ -0,0 +1,155 @@
+                let junctionLayers = sourcesByType[.junction]!.map { AGSFeatureLayer(featureTable: $0.featureTable) }
+                operationalLayers?.addObjects(from: junctionLayers)
+                
+                // Create a renderer for the associations.
+                let attachmentValue = AGSUniqueValue(description: "Attachment", label: "", symbol: self.attachmentSymbol, values: [AGSUtilityAssociationType.attachment])
+                let connectivityValue = AGSUniqueValue(description: "Connectivity", label: "", symbol: self.connectivitySymbol, values: [AGSUtilityAssociationType.connectivity])
+                self.associationsOverlay.renderer = AGSUniqueValueRenderer(fieldNames: ["AssociationType"], uniqueValues: [attachmentValue, connectivityValue], defaultLabel: "", defaultSymbol: nil)
+                
+                self.createSwatches()
+                self.addAssociationGraphics()
+                self.mapViewDidChange()
+            }
+        }
+    }
+    
+    func addAssociationGraphics() {
+        // Check if the current viewpoint is outside of the max scale.
+        if let viewpoint = mapView.currentViewpoint(with: .centerAndScale),
+            viewpoint.targetScale >= maxScale {
+            return
+        }
+        
+        // Check if the current viewpoint has an extent.
+        if let viewpoint = mapView.currentViewpoint(with: .boundingGeometry) {
+            // Get all of the associations in extent of the viewpoint.
+            utilityNetwork.associations(withExtent: viewpoint.targetGeometry.extent) { [weak self] (associations, error) in
+                guard let self = self else { return }
+                if let error = error {
+                    print("Error loading associations: \(error)")
+                } else if let associations = associations {
+                    let graphics: [AGSGraphic] = associations.compactMap { association in
+                        // If it the current association does not exist, add it to the graphics overlay.
+                        let associationGID = association.globalID
+                        guard !self.associationsOverlay.graphics.contains(where: {
+                            ($0 as! AGSGraphic).attributes["GlobalId"] as? UUID == associationGID
+                            }) else {
+                                return nil
+                            }
+                        let symbol: AGSSymbol
+                        switch association.associationType {
+                        case .attachment:
+                            symbol = self.attachmentSymbol
+                        case .connectivity:
+                            symbol = self.connectivitySymbol
+                        default:
+                            return nil
+                        }
+                        return AGSGraphic(geometry: association.geometry, symbol: symbol, attributes: ["GlobalId": associationGID, "AssociationType": association.associationType])
```

Comment:
Re-indent to look better?
```swift
return AGSGraphic(
    geometry: association.geometry,
    symbol: symbol,
    attributes: [
        "GlobalId": associationGID,
        "AssociationType": association.associationType
])
```

Reply:
If you are to reindent, the preferred way is with the closing square bracket on its own line:

```suggestion
                        return AGSGraphic(
                            geometry: association.geometry,
                            symbol: symbol,
                            attributes: [
                                "GlobalId": associationGID,
                                "AssociationType": association.associationType
                            ]
                        )
```